ios/BlurAmount.m

```diff
@@ -1,54 +1,52 @@
 #import "BlurAmount.h"
 #import <objc/runtime.h>
 
-@interface UIBlurEffect (Protected)
-
-@property (nonatomic, readonly) id effectSettings;
-@property (nonatomic, copy, class) NSNumber *localBlurAmount;
-
-@end
-
-
-@implementation BlurAmount
-
-static NSNumber *_localBlurAmount = nil;
-
-+ (instancetype)effectWithStyle:(UIBlurEffectStyle)style
-{
-    id result = [super effectWithStyle:style];
-    object_setClass(result, self);
-
-    return result;
+@implementation UIBlurEffect (RNBlur)
+
+NSNumber *localBlurAmount;
+
++ (id)invokeOriginalMethod:(id)target selector:(SEL)selector {
```

Comment:
I'm pretty 👎 on using swizzling and categories to get behavior. It's fragile and hard to understand.